### PR TITLE
Fix `--no-verify` flag.

### PR DESCRIPTION
**Pull request by lepiolet**
It was not working. When it was passed the certificate was checked anyway.

### CODE REVIEW

File: appgate/appgate.py [hunk 1]
```diff
@@ -112,12 +112,12 @@ def get_context(args: OperatorArguments,
     dry_run_mode = os.getenv(DRY_RUN_ENV) or ('1' if args.dry_run else '0')
     cleanup_mode = os.getenv(CLEANUP_ENV) or ('1' if args.cleanup else '0')
     spec_directory = os.getenv(SPEC_DIR_ENV) or args.spec_directory or SPEC_DIR
-    no_verify = os.getenv(APPGATE_SSL_NO_VERIFY) or '0'
+    no_verify = (os.getenv(APPGATE_SSL_NO_VERIFY) or '0') == '1' or args.no_verify
```
Comment (thomascellerier) [inline appgate/appgate.py:115]: You can pass '0' as a second argument to getenv instead https://docs.python.org/3/library/os.html#os.getenv

File: appgate/appgate.py [hunk 2]
```diff
@@ -112,12 +112,12 @@ def get_context(args: OperatorArguments,
...
     appgate_cacert = os.getenv(APPGATE_SSL_CACERT)
     appgate_cacert_path = None
-    if no_verify != '1' and appgate_cacert:
+    if not no_verify and appgate_cacert:
```
Comment (thomascellerier) [inline appgate/appgate.py:118]: Double negatives, maybe assign an intermediate variable like:
```Python
verify = not no_verify
```

**Review by thomascellerier — APPROVED** [review]

File: appgate/appgate.py (hunk 1 above)
Comment (lepiolet) [reply appgate/appgate.py:115]: ahh yes.

File: appgate/appgate.py (hunk 2 above)
Comment (lepiolet) [reply appgate/appgate.py:118]: yeah, the `not no` looks bad :D

**Pull request merged**